CLAUDE.md

```
@@ -767,6 +770,95 @@ dclaude
+
+```bash
+dclaude aws login                      # Default profile
+dclaude aws login --profile staging    # Specific profile
+```
+
+Uses `aws login` (not `aws sso login`) — this is the more universal command that works for both IAM console users and SSO users. All arguments are forwarded to `aws login` in the container.
+
+### Technical Implementation
+
+**Config precedence:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)